src/test/java/com/hocs/test/pages/foi/FOIProgressCase.java

```diff
@@ -24,11 +24,16 @@
     Documents documents;
 
     public void moveCaseFromCaseCreationToAllocation() {
+        String responseDate = getTodaysDate();
         clickTheButton("Confirm");
-        caseCreationStage.selectValidityOfRequest("Yes");
+        caseCreationStage.selectValidityOfRequest("Valid");
         safeClickOn(continueButton);
         waitABit(250);
+        documents.addInitialResponseDocument();
+        responseDate = getDatePlusMinusNDaysAgo(-10);
+        recordCaseData.enterDateIntoDateFieldsWithHeading(responseDate, "When was the acknowledgement response to the valid request issued?");
```

Comment:
Same  as comment above, these 2 lines should be in a page method which we call here.